ensemble_profiler/server.py

```diff
@@ -111,7 +111,12 @@ def __init__(self, address, actor_handles,
             raise ValueError("Multiple Patients specified."
                              "Specify only one.")
         patient_name = patient_name[0]
-        prediction_tensor = torch.zeros((1, 1, PREDITICATE_INTERVAL))
+
+        obs_w_30sec = query_kwargs.pop("obs_w_30sec", None)
```

Comment:
why obs_window has to be per query basis? Can it part of http server instantiation?

Reply:
I assume that when profile client make request it can change the obs_window.
If we put it as part of http server instantiation then the client will always have to send the same obs_window.
For example, when the patients are on stable condition client will only look at interval of obs_window 30s, however when the patients are critical condition, the client can send obs_window 30s, 2min, 5 min, etc

Reply:
Yes the client has to be consistent in sending the information. Any reasons for the assumption you are making?